pkg/operator/options/options.go

```diff
@@ -40,13 +40,15 @@ type Options struct {
 	ClusterID               string
 	ClusterCNI              string
 	VMMemoryOverheadPercent float64
+	Interruption            bool
 }
 
 func (o *Options) AddFlags(fs *coreoptions.FlagSet) {
 	fs.StringVar(&o.ClusterName, "cluster-name", env.WithDefaultString("CLUSTER_NAME", ""), "[REQUIRED] The kubernetes cluster name for resource discovery.")
 	fs.StringVar(&o.ClusterID, "cluster-id", env.WithDefaultString("CLUSTER_ID", ""), "The external kubernetes cluster id for new nodes to connect with.")
 	fs.StringVar(&o.ClusterCNI, "cluster-cni", env.WithDefaultString("CLUSTER_CNI", "terway-eniip"), "The network cni used by the cluster.")
 	fs.Float64Var(&o.VMMemoryOverheadPercent, "vm-memory-overhead-percent", utils.WithDefaultFloat64("VM_MEMORY_OVERHEAD_PERCENT", 0.075), "The VM memory overhead as a percent that will be subtracted from the total memory for all instance types.")
+	fs.BoolVar(&o.Interruption, "interruption", env.WithDefaultBool("INTERRUPTION", false), "Enable interruption handling.")
```

Comment:
Let's enable it by default, AWS doesn't enable because it needs special SQS configuration.